src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -34,96 +34,53 @@
   <div class="card-block row">
     <div class="card-text">
       {{ form_errors(categoryForm) }}
-
-      {{ ps.form_group_row(categoryForm.name, {}, {
-        'label': 'Name'|trans({}, 'Admin.Global'),
-        'help': 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}'
-      }) }}
-
-      <div class="form-group row">
-        {{ ps.label_with_help(('Displayed'|trans({}, 'Admin.Global')), ('Click on "%displayed_label%" to index the category on your shop.'|trans({'%displayed_label%': 'Displayed'|trans({}, 'Admin.Global')}, 'Admin.Catalog.Help'))) }}
-        <div class="col-sm">
-          {{ form_widget(categoryForm.active) }}
-          <small class="form-text">
-            {{ 'If you want a category to appear in the menu of your shop, go to [1]Modules > Module Manager[/1] and configure your menu module.'|trans({
-              '[1]': '<a href="' ~ path('admin_module_manage') ~ '" target="_blank" rel="noopener noreferrer nofollow">',
-              '[/1]': '</a>'
-            }, 'Admin.Catalog.Help')|raw }}
-          </small>
-        </div>
-      </div>
-
+      {{ form_row(categoryForm.name) }}
```

Comment:
The purpose of this simplified form is to simply have to use
`{{ form_row(categoryForm.name) }}`

Why do you need to display each field manually here? Is it because of the thumbnail special design? If so they should be customized either in the category form theme or maybe even in the UI kit if such display needs to be reused somewhere else

Reply:
Don't get me wrong it's already a lot more simplified, but I wonder what stops us to go to the whole thing